vic/drivers/cesm/bld/vic.buildexe.csh

```diff
@@ -0,0 +1,23 @@
+#! /bin/csh -f
+
+cd $OBJROOT/lnd/obj
+
+set comp = 'unknown'
+if ($COMP_INTERFACE == 'MCT' ) set comp = mct
+if ($COMP_INTERFACE == 'ESMF') set comp = esmf
+
+cat >! Filepath << EOF
+$CASEROOT/SourceMods/src.vic
+$CODEROOT/lnd/vic/vic
+$CODEROOT/lnd/vic/vic/vic_run/src
+$CODEROOT/lnd/vic/vic/vic_run/include
+$CODEROOT/lnd/vic/vic/drivers/shared/src
+$CODEROOT/lnd/vic/vic/drivers/shared/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/src
+$CODEROOT/lnd/vic/vic/drivers/cesm/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/cpl_$comp/
+EOF
+
+set vicdefs = ""
```

Comment:
@apcraig - 

What is the best way to set build time flags and options using the CESM Makefile?  Is this the line to do it in?

We are currently setting some basic C compiler flags: `CFLAGS  =  -ggdb -O0 -Wall -Wextra -fPIC`

and a build time log level flag: `-DLOG_LVL=$(LOG_LVL)` where `$(LOG_LVL)` is an integer that sets the verbosity of logging within VIC.